packages/session-replay-browser/src/config/types.ts

```diff
@@ -31,6 +31,11 @@ export type PrivacyConfig = {
   unmaskSelector?: string[];
 };
 
+export type InteractionConfig = {
+  trackEveryNms?: number;
+  enabled?: boolean; // defaults to false
```

Comment:
If it has a default, should it not be marked as optional?

Reply:
Yeah I think now it makes sense to mark it as non optional since it defaults to `false`. I had a previous iteration where it was defaulted to `true` and I wanted to make sure if it wasn't specified it'd be `true`.